price-feeder/oracle/provider/provider.go

```diff
@@ -24,6 +24,7 @@ var ping = []byte("ping")
 type Provider interface {
```

Comment:
Not that you have to do it at all or even in this PR, but really these methods in the interface need to have godocs so we know what they do. Is that something you think you can tackle in this PR? They don't have to be super elaborate either.

Reply:
Sure ;)
done at https://github.com/umee-network/umee/pull/649/commits/e86167c878e13e602495f07958ea2bf73432f419